package.json

```diff
@@ -18,7 +18,7 @@
     "translations:generate": "node generate_translations.mjs",
     "translations:init": "git submodule init && pnpm translations:update",
     "translations:update": "git submodule update --remote --recursive",
-    "tailwind": "tailwindcss -i ./src/style.css -o ./dist/styles/styles.css --minify",
+    "tailwind": "tailwindcss -i ./src/style.css -o ./dist/styles/styles.css",
```

Comment:
Probably should keep minify.

Reply:
Problem is that it seems to run minify in development also, so if you make a change and add a css class, it is missing until you rerun `pnpm start`. So it should be changed to run minify only in prod.

Reply:
No that seems to be caused by something else, added minify back.